.github/workflows/tests.yaml

```diff
@@ -11,8 +11,13 @@ on:
 
 jobs:
   build_tensorflow_autotools:
-    runs-on: ubuntu-18.04
-    container: ridgerun/r2inference:v0.1.4
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+       os: [ubuntu-18.04]
```

Comment:
Shouldn't we add ubuntu-16.04 too?

Reply:
I don't see why that would be needed because the jobs will run in the containers (which already test Ubuntu 16.04 and Ubuntu 18.04 environments).

Reply:
This is absorbed in the `container` matrix. Not on the `os` matrix. This only selects the OS where the container will run on. I recommend removing this from the matrix.

Reply:
Agreed!